x:
  k: 1.0
  lo: -0.4
  hi: 0.4
  vpid:
    kp: 100.0
    ki: 50.0
    kd: 5.0
    kf: 0.2
    lo: -30.0
    hi: 30.0
y:
  k: 1.0
  lo: -0.3
  hi: 0.3
  vpid:
    kp: 100.0
    ki: 50.0
    kd: 3.0
    kf: 0.2
    lo: -30.0
    hi: 30.0
z:
  k: 0.3
  lo: -0.3
  hi: 0.3
  vpid:
    kp: 100.0
    ki: 40.0
    kd: 10.0
    kf: 0.2
    lo: -30.0
    hi: 30.0
    offset: 8.0
roll:
  k: 2.0
  lo: -25.0
  hi: 25.0
  vpid:
    kp: 0.34
    ki: 0.012
    kd: 0.0004
    kf: 0.2
    lo: -30.0
    hi: 30.0
    roll_scale_factor: 0
yaw:
  k: 1.0
  lo: -40.0
  hi: 40.0
  vpid:
    kp: 0.35
    ki: 0.07
    kd: 0.02
    kf: 0.2
    lo: -30.0
    hi: 30.0
